packages/tasit-action/src/Contract.test.js

```diff
@@ -417,35 +416,37 @@ describe("TasitAction.Contract", () => {
       const listener1 = message => {};
       const listener2 = message => {};
 
-      expect(txSubscription.subscribedEventNames()).to.be.empty;
+      expect(txSubscription.subscribedEventNames()).to.not.include(
+        "confirmation"
+      );
 
       txSubscription.on("confirmation", listener1);
 
       expect(() => {
         txSubscription.on("confirmation", listener2);
       }).to.throw();
 
-      expect(txSubscription.subscribedEventNames()).to.deep.equal([
-        "confirmation",
-      ]);
+      expect(txSubscription.subscribedEventNames()).to.include("confirmation");
```

Comment:
This test seems too flexible. Did you change it to this because it includes "error" as well right now?

Reply:
Yes

Reply:
Ah ok. I think I would prefer explicitly looking for both error and confirmation then, as opposed to just having it include confirmation.

Reply:
Oh, my fault. I've just replaced all `includes` to `deep.equal`.

Reply:
Thanks. Resolving this.